pkg/exec_bash_cmd.go

```diff
@@ -1,26 +1,43 @@
-//go:build !windows
-
 package pkg
 
 import (
+	"bytes"
 	"fmt"
 	"io"
-	"log"
 	"os"
 	"os/exec"
 	"os/signal"
 	"strings"
 	"syscall"
 
 	"github.com/creack/pty"
+	"github.com/pterm/pterm"
 	"golang.org/x/term"
 )
 
-func ExecBashCmd(dir string, name string, arg ...string) {
+func ExecBashCmd(runtime_os string, dir string, name string, arg ...string) string {
+	if runtime_os == "windows" {
```

Comment:
confused on how I specify OS in dlp-cli command @dwu359? can you add that to the readme docs

Reply:
-w